rust/types/src/collection.rs

```diff
@@ -50,25 +49,33 @@ impl std::fmt::Display for CollectionUuid {
     }
 }
 
-#[derive(Clone, Debug, Deserialize, PartialEq, Serialize, ToSchema)]
+#[derive(Clone, Debug, Deserialize, PartialEq, Serialize, ToSchema, bon::Builder)]
```

Comment:
added a crate to generate builder helpers
makes it much easier to construct Collections in tests and helps document exactly what fields any given test cares about